scipy/optimize/_basinhopping.py

```diff
@@ -367,55 +367,55 @@ def basinhopping(func, x0, niter=100, T=1.0, stepsize=0.5,
     ----------
     func : callable ``f(x, *args)``
         Function to be optimized.  ``args`` can be passed as an optional item
-        in the dict ``minimizer_kwargs``
+        in the dict `minimizer_kwargs`
```

Comment:
Did you mean to change this?  everywhere else the docs are using two tics. I'm not sure whether one or two tics is preferred, but if we switch to one tic we should probably do it everywhere

Reply:
you switched to single tics several other places also

Reply:
Yes this is intended as it's our convention.

Reply:
@js850 Please see the standards in https://numpydoc.readthedocs.io/en/latest/format.html. They are ambiguous, but because `minimizer_kwargs` is a keyword of the function, it belongs in single backticks. Double backticks are for things to be rendered as code that are not simply 1) the name of a SciPy function/method/class or 2) a keyword of the current function/method/class. There are other issues where this is debated (e.g. all of it should render as code), but @tupui is making this consistent with the convention throughout `stats`.